### PR TITLE
See all reminders

### PR DESCRIPTION
**Pull request by maiastone**
## Purpose

Renders the list of reminders in the root url.  Addresses issue #1 

## Approach

Creating a route of reminders allows us to access the reminder model.  The reminder model is able to access the reminders from the data store.  The model object is setup to parallel the data in the store.  The router is configured to render the reminders.hbs as a child of application, therefor showing up to the user in the root url.  The reminders.hbs uses the each method to render a list item of reminder title for each reminder.

### Learning

We primarily relied on the Ember docs, but also referred to the following article.
[An In-Depth Introduction To Ember.js](https://www.smashingmagazine.com/2013/11/an-in-depth-introduction-to-ember-js/)

### Test coverage 

There is a passing acceptance test to assert that user views the reminders on the homepage.

### Follow-up tasks

- [Issue #2 ](https://github.com/turingschool-projects/1608-remember-5/issues/2)

#### After

image

@martensonbj @tman22 @nfosterky

### CODE REVIEW

File: app/models/reminder.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  title: DS.attr('string'),
+  date: DS.attr('date'),
+  notes: DS.attr('string'),
+  pinned: false
```
Comment (nfosterky) [inline app/models/reminder.js]: Please add a attribute value for pinned and default it to false
https://guides.emberjs.com/v2.4.0/models/defining-models/#toc_options

Comment (maiastone) [inline app/models/reminder.js]: Done!  Thanks for the feedback.

**Comment by maiastone** [comment]
Please review the additional passing acceptance test.

**Comment by maiastone** [comment]
Closes Issue #1

**Pull request merged**